bahmni-reports/package/docker/scripts/docker_publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+REPORTS_IMAGE_TAG=$(cat bahmni-emr/.appversion)-${GITHUB_RUN_NUMBER}
+echo ${DOCKER_HUB_TOKEN} | docker login -u ${DOCKER_HUB_USERNAME} --password-stdin
+echo "[INFO] Pushing build images"
+docker push bahmniindiadistro/reports:${REPORTS_IMAGE_TAG}
+
+echo "[INFO] Tagging build images as SNAPSHOT Images"
+REPORTS_SNAPSHOT_IMAGE_TAG=$(cat bahmni-emr/.appversion)-SNAPSHOT
+docker tag bahmniindiadistro/reports:${REPORTS_IMAGE_TAG} bahmniindiadistro/reports:${REPORTS_SNAPSHOT_IMAGE_TAG}
+
+echo "[INFO] Pushing SNAPSHOT images"
+docker push bahmniindiadistro/reports:${REPORTS_SNAPSHOT_IMAGE_TAG}
+
+docker logout
```

Comment:
You are using GitHub actions for docker build and push. You don't need this shell

Reply:
removed the script